examples/release_highlights/plot_release_highlights_0_24_0.py

```diff
@@ -0,0 +1,225 @@
+# flake8: noqa
+"""
+========================================
+Release Highlights for scikit-learn 0.24
+========================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 0.24! Many bug fixes
+and improvements were added, as well as some new key features. We detail
+below a few of the major features of this release. **For an exhaustive list of
+all the changes**, please refer to the :ref:`release notes <changes_0_24>`.
+
+To install the latest version (with pip)::
+
+    pip install --upgrade scikit-learn
+
+or with conda::
+
+    conda install -c conda-forge scikit-learn
+
+.. meta::
+   :keywords: highlights
+
+"""
+
+##############################################################################
+# Searching parameter space with successive halving
+# -------------------------------------------------
+# Successive halving, a state of the art method, is now available to
+# explore the space of the parameters and identify their best combination.
+# It is still experimental, in order to import it `enable_halving_search_cv`
+# should be imported from `experimental`.
```

Comment:
No need to detail how to import it in the highlights.

```suggestion
# It is still experimental.
```

I'd suggest noting that it's experimental in a `.. note::` but it's fine as-is too.